modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/MDFRequestForm/steps/Review/components/Body/components/ActivityReviewEntry/ActivityReviewEntry.tsx

```diff
@@ -0,0 +1,92 @@
+/**
```

Comment:
@DiogoNo, we need to avoid to put complex hooks inside a component that repeat a lot.